docs/docs/glossary.md

```diff
@@ -3,21 +3,25 @@ title: Glossary
 tags: [protocol, glossary]
 ---
 
+### Aztec
+
+import { Glossary } from '@site/src/components/Snippets/glossary_snippets';
+
 ### `aztec-nargo`
 
-The command line tool used to compile Aztec contracts. It gets its name from the Noir compiler, `nargo`. `aztec-nargo` is really just a specific version of `nargo`. You can read more about `nargo` [here](#nargo).
+<Glossary.CLI.aztec_nargo />
+
+You can read more about `nargo` [here](#nargo).
 
 ### Aztec.nr
 
-A [Noir](https://noir-lang.org) framework for writing smart contracts on Aztec.
+<Glossary.Libs.aztec_nr />
 
 Read more and review the source code [here](https://aztec.nr).
 
```

Comment:
Add a line for `aztec-wallet`

Reply:
"Aztec Wallet" definition now after "Aztec"